src/style.css

```
@@ -1 +1,44 @@
-/* add styles here */
+
+body {
+  display: block;
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+ width: 16px;
+ height: 16px;
+ background-repeat: no-repeat;
+ margin-right: 4px;
+ background-image: url(./images/star.svg);
+ background-position: 50%;
+}
+
+
+.stars--1 .stars__star:nth-child(-n +1) {
+background-image: url(./images/star-active.svg);
+}
+
+.stars--2 .stars__star:nth-child(-n +2) {
+  background-image: url(./images/star-active.svg);
+
+}
```

```suggestion
.stars--1 .stars__star:nth-child(-n +1),
.stars--2 .stars__star:nth-child(-n +2) {
  background-image: url(./images/star-active.svg);
}
```
combine it using the comma